modules/common-fs2/src/main/scala/com/snowplowanalytics/snowplow/enrich/common/fs2/Enrich.scala

```diff
@@ -290,10 +290,7 @@ object Enrich {
     val asStrE = customOutputFormat match {
       case None => Right(tsv)
       case Some(outputFormat) =>
-        val jsonE = com.snowplowanalytics.snowplow.analytics.scalasdk.Event
-          .parse(tsv)
-          .map(_.toJson(lossy = true))
-          .toEither
+        val jsonE = EnrichUtils.transformTsvToJson(tsv)
```

Comment:
Given the current state, I think the simplest solution would be to introduce a new output format like "BigQueryJson", then, this would be enabled from the configuration.

Reply:
where does `EnrichedEvent` comes from and what is the tsv used for?

it works like `EnrichedEvent` --> TSV --> `Event` ---> Json 

why do we need three transformations?

Reply:
so add support for format `BigQueryJson` next to `FlattenedJson` and `Eventbridge`? 

that is one way and would like to hear about my previous comment so I have a better picture

Reply:
There is a chance that we could skip the TSV parsing, or, go directly from `EnrichedEvent` to Json, still, I have not dig into that optimization yet.

TSV is the default output format, it is also attached to the JSON when using eventbridge.

EnrichedEvent is produced from the enricher rules

Reply:
@AlexITC please check now. I also added unit tests

Reply:
@AlexITC  please check now. I also added unit tests

Reply:
need to update documentation as well. how is that managed

Reply:
Documentation related, I'll update the release notes.